Jellyfin.SkiaSharp.NativeAssets.LinuxArm/cross-compile-libSkiaSharp.sh

```diff
@@ -4,7 +4,7 @@ TOOLCHAIN_ARM64=gcc-linaro-4.9.4-2017.01-x86_64_aarch64-linux-gnu
 
 set -e
 if [ ! -d "skia" ] ; then
-    git clone https://github.com/mono/skia.git -b v1.68.0 --depth 1 skia
+    git clone https://github.com/mono/skia.git -b v1.68.1 --depth 1 skia
```

Comment:
The latest version is `v1.68.1.1` I think we should use that?

Reply:
Well jellyfin uses currently `1.68.1`, see:

https://github.com/jellyfin/jellyfin/blob/b3811a9498fe06b68693f4a269de902cdd7eb2a2/Jellyfin.Drawing.Skia/Jellyfin.Drawing.Skia.csproj#L15

So I feel like it's safer to perfectly match. But we could also upgrade everywhere to `1.68.1.1`.

Reply:
Ah okay, yeah I see. Someone else would have to make that call I'm not familiar with this project, probably @EraYaN

Reply:
Ooh building this was quite a pain. I need to dig a bit to see which builds are still required. (Upstream also added some I think)

Reply:
I'm currently compiling using the bash script but I'm using WSL on windows and it's taking hours... but so far so good.

Reply:
Well after hours... it finally failed:
```
[31/1043] compile ../../src/ports/SkFontConfigInterface_direct_factory.cpp
FAILED: obj/src/ports/fontmgr_fontconfig.SkFontConfigInterface_direct_factory.o
arm-linux-gnueabihf-g++ -MMD -MF obj/src/ports/fontmgr_fontconfig.SkFontConfigInterface_direct_factory.o.d -DNDEBUG -DSK_SAMPLES_FOR_X -DSK_GAMMA_APPLY_TO_A8 -DSK_ALLOW_STATIC_GLOBAL_INITIALIZERS=0 -DSK_ENABLE_DISCRETE_GPU -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -march=armv7-a -mfpu=neon -mthumb -O3 -fdata-sections -ffunction-sections -Wno-unused-parameter -DSKIA_C_DLL -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -c ../../src/ports/SkFontConfigInterface_direct_factory.cpp -o obj/src/ports/fontmgr_fontconfig.SkFontConfigInterface_direct_factory.o
In file included from ../../src/ports/SkFontConfigInterface_direct_factory.cpp:8:0:
../../src/ports/SkFontConfigInterface_direct.h:14:35: fatal error: fontconfig/fontconfig.h: No such file or directory
 #include <fontconfig/fontconfig.h>
                                   ^
```

Reply:
(seems related to https://github.com/mono/SkiaSharp/issues/633)

Reply:
Ok well I finally made it to compile by downloading `libfontconfig` manually (both includes and libs for arm).

It depends on your disto but for Ubuntu I add to extract the .deb manually:
```
wget http://ports.ubuntu.com/pool/main/f/fontconfig/libfontconfig1-dev_2.11.94-0ubuntu1_armhf.deb
ar x libfontconfig1-dev_2.11.94-0ubuntu1_armhf.deb
tar xvf data.tar.xz
```

And copying files to `tools/arm-bcm2708/arm-linux-gnueabihf/arm-linux-gnueabihf/` folder.

@EraYaN  A few remarks:
- I'm not sure why at the end of the script you compile for x64 too
- Would be great to run the script through a docker container to have a reproductible build